templates/view.html

```diff
@@ -75,10 +75,10 @@
                 <b>{{ call $.T "report_type" }}:</b>
                 <form method="post" action="/report/{{$.Torrent.ID}}">
                     {{ block "csrf_field" $ }}{{end}}
-                    <input type="radio" name="report_type" value="illegal" id="illegal" required> <label for="illegal">{{ call $.T "illegal_content" }}</label><br />
-                    <input type="radio" name="report_type" value="spam" id="spam" required> <label for="spam">{{ call $.T "spam_garbage" }}</label><br />
-                    <input type="radio" name="report_type" value="wrongcat" id="wrongcat" required> <label for="wrongcat">{{ call $.T "wrong_category" }}</label><br />
-                    <input type="radio" name="report_type" value="dup" id="dup" required> <label for="dup">{{ call $.T "duplicate_deprecated" }}</label><br />
+                    <input type="radio" name="report_type" value="illegal_content" id="illegal" required> <label for="illegal">{{ call $.T "illegal_content" }}</label><br />
+                    <input type="radio" name="report_type" value="spam_garbage" id="spam" required> <label for="spam">{{ call $.T "spam_garbage" }}</label><br />
```

Comment:
Anyway, if you've tested it you should see what I mean and how it works. getReportDescription is just a function that will disappear because it's a fix for old report description in the database